bots/images/scripts/fedora-atomic.bootstrap

```diff
@@ -10,4 +10,4 @@ BASE=$(dirname $0)
 # The Fedora URLs have the version twice in the name. for example:
 # https://dl.fedoraproject.org/pub/alt/atomic/stable/Fedora-Atomic-25-20160921.0/CloudImages/x86_64/images/Fedora-Cloud-Base-25-20160921.0.x86_64.qcow2
 $BASE/atomic.bootstrap "$1" "$url" \
-    "Fedora-Atomic-25-[0-9\.]+" "CloudImages" "x86_64" "images" "Fedora-Atomic-25-[0-9\.]+.x86_64.qcow2"
+    "Fedora-Atomic-[-0-9\.]+" "CloudImages" "x86_64" "images" "Fedora-Atomic-[-0-9\.]+.x86_64.qcow2"
```

Comment:
Lastly, as modularity comes online, and if Atomic Host moves to a single stream, it's likely that a specific modularized buildroot for atomic host will be necessary. It will no longer be the case that a specific version of Fedora is appropriate as a buildroot. 

This is longer term, but this commit looks toward that.